components/compliance-service/api/jobs/server/server.go

```diff
@@ -106,12 +106,13 @@ func (srv *Server) Create(ctx context.Context, in *jobs.Job) (*jobs.Id, error) {
 		return nil, status.Error(codes.InvalidArgument, "Invalid job: nodes or node selectors required.")
 	}
 
-	sID, err := srv.db.AddJob(in)
+	sID, name, err := srv.db.AddJob(in)
```

Comment:
I'm a little confused about why we need to change the signature of AddJob.  `in` is a `jobs.Job` and should already have the Name assigned to it. I don't see anything in AddJob that mutates that name.

I can see some value in being defensive here since AddJob mutates `Id` we should indeed maybe assume it mutates everything. But, in that case, I think one of the following signatures for AddJob might be preferable:

1) `AddJob(inJob jobs.Job) (jobs.Job, error)` (i.e take the input by value return the an entire job object)
2) `AddJob(inJob *jobs.Job) error` (i.e. take the input by reference, assume AddJob mutates it appropriately)

If I'm right about not actually needing this `name` return, it might be best to leave AddJob as-is for now to reduce the scope of this change.

Reply:
@stevendanna , I have made changes for this, please have a look.